File: library/src/main/java/su/levenetc/android/textsurface/interfaces/ISet.java
package su.levenetc.android.textsurface.interfaces;

import java.util.LinkedList;

import su.levenetc.android.textsurface.contants.TYPE;

/**
 * Created by Eugene Levenetc.
 */
public interface ISet extends ISurfaceAnimation {
	TYPE getType();

	LinkedList<ISurfaceAnimation> getAnimations();
}

File: library/src/main/java/su/levenetc/android/textsurface/interfaces/ISet.kt
package su.levenetc.android.textsurface.interfaces

import su.levenetc.android.textsurface.contants.TYPE
import java.util.*

/**
 * Created by Eugene Levenetc.
 */
interface ISet : ISurfaceAnimation {
    val type: TYPE
    val animations: LinkedList<ISurfaceAnimation>
}